### PR TITLE
A link to the commit currently running on the server

### PR DESCRIPTION
**Pull request by henrikssn**
Adds a little icon in the topbar which shows the commit the server is running. Nice to ensure that the auto-pull works :)

One question inline.

### CODE REVIEW

File: app/views/layouts/_topbar.html.erb [hunk 1]
```diff
@@ -1,4 +1,11 @@
-        <ul class="loginbar pull-right">
+<ul class="loginbar pull-right">
+  <%if user_signed_in? && current_user.admin?%>
+    <% commit = `git rev-parse HEAD`[0,6] %>
```
Comment (henrikssn) [inline app/views/layouts/_topbar.html.erb]: I know this probably should not be in the view, but where should I add this? In the menus controller?

Comment (Zofz) [inline app/views/layouts/_topbar.html.erb]: I think that we should implement some kind of admin-controller that could be used for things like this. Until then, the menu controller seems like a sane choice, or leave it in the view (dirty).

**Comment by Zofz** [comment]
What is the difference between Travis Push or Pull Req.?

**Comment by henrikssn** [comment]
I think push means this branch and PR is after merge into master.

Problem was that the test didn't pass on master. It is fixed now.

**Comment by henrikssn** [comment]
@Zofz Typ så här? Jag fick det inte att funka dock, @commit / @commit_url finns inte i viewen

**Comment by Zofz** [comment]
@henrikssn Juste, du måste lägga det i application-controller, på samma ställe som menyerna laddas.
Annars körs det ju inte varje gång, utan bara när man är inne i den faktiska meny-controllern.

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -42,4 +43,8 @@ def set_locale
       I18n.locale = locale
       redirect_to(:back) if params[:locale]
     end
+    def get_commit
+      @commit = `git rev-parse HEAD`[0,6]
```
Comment (Zofz) [inline app/controllers/application_controller.rb]: Looking good 😄

**Comment by henrikssn** [comment]
@Zofz I see, tack! Är vi nöjda med denna nu?

**Comment by Zofz** [comment]
@henrikssn Det tycker jag definitivt 😄

**Comment by Zofz** [comment]
Passing everything, will you merge?

**Comment by Zofz** [comment]
Very nice! 😄

**Pull request merged**